lib/dolly/request.rb

```diff
@@ -40,6 +40,11 @@ def delete resource
       request :delete, full_path(resource), {}
     end
 
+    def attach resource, attachment_name, data, headers = {}
+      data = StringIO.new(data) if data.is_a?(String)
```

Comment:
@netlogics yes I think we should support inline as well, just not for this PR. Ill update the PR title